packages/experimental/Link/src/Link.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { linkName, LinkType, LinkProps, ILinkState } from './Link.types';
+import { compose, mergeProps, withSlots, UseSlots } from '@fluentui-react-native/framework';
+import { useLink } from './useLink';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+import { stylingSettings } from './Link.styling';
+
+/**
+ * A function which determines if a set of styles should be applied to the component given the current state and props of the link.
+ *
+ * @param layer The name of the state that is being checked for
+ * @param state The current state of the link
+ * @param userProps The props that were passed into the link
+ * @returns Whether the styles that are assigned to the layer should be applied to the link
+ */
+export const linkLookup = (layer: string, state: ILinkState, userProps: LinkProps): boolean => {
+  return (
+    state[layer] ||
+    userProps[layer] ||
+    layer === userProps['appearance'] ||
+    (layer === 'inline' && userProps.inline) ||
```

Comment:
nit: the `userProps[layer]` should take care of the inline and disabled lines in this function.